libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -332,6 +355,13 @@ export class SkyDateRangePickerComponent
   }
 
   public ngOnInit(): void {
+    this.#ngControl = this.#injector.get(NgControl, null, {
+      optional: true,
```

Comment:
I did this because we do have a couple unit tests that test the control without a form control or ngModel attached. I am a little worried this might be a breaking change if we don't make this optional (as much as I would like to just make it required here).